components/wifi.rst

```diff
@@ -85,6 +85,9 @@ Configuration variables:
   The downside is that this option connects to the first network the ESP sees, even if that network is very far away and
   better ones are available.
 
+- **enable_btm** (*Optional*, bool): Only on ``esp32`` with ``esp-idf``. Enable 802.11v BSS Transition Management support.
+- **enable_rrm** (*Optional*, bool): Only on ``esp32`` with ``esp-idf``. Enable 802.11k Radio Resource Management support.
```

Comment:
Quick check, does enabling these have any downsides? If not, can we enable them per default? Would also save adding a new config option.

Reply:
Good question ... I've seen too many WiFi compatibility issues to feel good with forcing something onto users, so I prefer having a config option.